deeplabcut/gui/tabs/extract_frames.py

```diff
@@ -197,6 +197,13 @@ def extract_frames(self):
         mode = self.extraction_method_widget.currentText()
         if mode == "manual":
             videos = list(self.video_selection_widget.files)
+            if not videos:
+                QtWidgets.QMessageBox.critical(
+                    self,
+                    "Error",
+                    "Please select at least one video to extract frames from.",
```

Comment:
I would change the wording here to "... exactly one ..." as only we will only open frame extraction for one video.

We could print a warning when more than 1 video is selected (saying "only the first video ({video-name}) will be opened to manually extract frames from")